packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -671,170 +671,170 @@ export default {
     display: flex;
 
     &:focus {
-        outline-color: $nav-link-focus-color;
-        border-radius: $nav-focus-borderRadius;
+        outline-color: common.$nav-link-focus-color;
+        border-radius: common.$nav-focus-borderRadius;
     }
 
-    @include media('>mid') {
-        margin-top: spacing(d);
-        margin-bottom: spacing(d);
+    @include f.media('>mid') {
+        margin-top: f.spacing(d);
+        margin-bottom: f.spacing(d);
     }
 
-    @include media('<=mid') {
+    @include f.media('<=mid') {
         width: 100%;
         text-align: left;
         display: block;
-        padding: spacing(c) 0;
+        padding: f.spacing(c) 0;
 
         &:focus {
-            outline-color: $nav-link-focus-color;
+            outline-color: common.$nav-link-focus-color;
             border-radius: 0;
         }
 
         &:hover {
-            background: $color-container-subtle;
+            background: f.$color-container-subtle;
             border-radius: 0;
         }
     }
 }
 
 .c-nav-list-btn-text {
-    @include font-size($nav-text-size);
-    color: $nav-text-color;
-    font-weight: $font-weight-regular;
-    @include media('>mid') {
-        font-weight: $nav-text-weight;
+    @include f.font-size(common.$nav-text-size);
+    color: common.$nav-text-color;
+    font-weight: f.$font-weight-regular;
+    @include f.media('>mid') {
+        font-weight: common.$nav-text-weight;
     }
 }
 
 .c-nav-list-btn--hoverAboveMid {
-    @include media('>mid') {
+    @include f.media('>mid') {
         &:hover {
-            background: $color-container-subtle;
-            border-radius: $nav-focus-borderRadius;
+            background: f.$color-container-subtle;
+            border-radius: common.$nav-focus-borderRadius;
         }
     }
 }
 
 .c-nav-list-text-sub {
     display: block;
     overflow: hidden;
-    @include font-size($nav-text-size);
-    margin-left: spacing(h);
-    color: $nav-text-color;
+    @include f.font-size(common.$nav-text-size);
+    margin-left: f.spacing(h);
+    color: common.$nav-text-color;
     &.u-showBelowMid {
-        @include media('>mid') {
+        @include f.media('>mid') {
             display: none !important;
         }
     }
 }
 
 .c-nav-featureLink {
-    width: $nav-featureLinkIcon-width;
-    height: $nav-featureLinkIcon-height;
+    width: common.$nav-featureLinkIcon-width;
+    height: common.$nav-featureLinkIcon-height;
 
     & path {
-        fill: $nav-icon-color;
+        fill: common.$nav-icon-color;
     }
 
-    @include media('<=mid') {
+    @include f.media('<=mid') {
         position: absolute;
         top: 0;
         right: 0;
-        width: spacing(d) + $nav-featureLinkIcon-width + spacing(d); // includes padding on both sides
-        height: spacing(d) + $nav-featureLinkIcon-height + spacing(d);
-        padding: spacing(d);
+        width: f.spacing(d) + common.$nav-featureLinkIcon-width + f.spacing(d); // includes padding on both sides
```

Comment:
I'm hesitant to ask you to change this but maybe this can just be `spacing(f) + common.$nav-featureLinkIcon-width` instead of the double `spacing(d)` same for height?